src/game/Pet.h

```diff
@@ -143,8 +143,8 @@ extern const uint32 LevelStartLoyalty[6];
 
 #define ACTIVE_SPELLS_MAX           4
 
-#define PET_FOLLOW_DIST  1.0f
-#define PET_FOLLOW_ANGLE (M_PI_F/2.0f)
+#define PET_FOLLOW_DIST  2.00f
```

Comment:
Let me freshen up your memory a little, then =) - Go load up classic, make a hunter and tame a pet and start moving - your pet follows you fine, but as soon as you stop, your pet positions itself directly west of you and turns around to stare at you.